### ISSUE TITLE
Puzzling error after upgrading to Python-3.12.x

### ISSUE THREAD

**Issue by oldpink** [nicfit/eyeD3#639]
After upgrading to Python version 3.12.7 from Python-3.11.10, can no longer use eyeD3, even after rebuilding/reinstalling the dependencies and doing the same with eyeD3 itself.
Whenever I true to run it, I get this error:

Plugin not found: classic

I tried looking for a patch (none I could locate), looked for anyone else having the same error, and even did a "git clone" of the latest Git version, and nothing turned up.
The only alternative that has at least enough of the same features to be usable in a pinch is Mutagen, but I still vastly prefer this to that.
Thanks

**Comment by nicfit** [comment]
I cannot reproduce this error @oldpink , and tests are passing (on HEAD) on Python 3.9-3.13.  Need more info.

**Comment by oldpink** [comment]
Yeah, for some reason I am now able to run the newest version of GPodder using Python-3.12.x.
It may be that the newer revisions of Python than the one I originally had problems corrected some hidden bug, or the newer version of GPodder corrected it.
Thanks for the reply.
btw...on a parallel note, I was able to overcome the problem of GPodder automatically seizing keyboard focus each time it updated by using the "Shade" function built into my window manager, which is WindowMaker.
It still updates with no problem, but the focus is at least left alone.
I thought I'd mention that for anyone else who has that as a problem.
